docs/Changelog.md

```diff
@@ -14188,3 +14188,61 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain input and output types to high-precision and 8 bit numeric tensors.</dd>
 </dl>
 
+### <a name="SoftmaxCrossEntropy-12"></a>**SoftmaxCrossEntropy-12**</a>
+
+  Loss function that measures the softmax cross entropy
+  between 'scores' and 'labels'.
+  The loss can be described as:
+      L = (l_1, l_2, ..., l_N), where N is the batch_size
+  
+  scores: (N, C) where C is the number of classes, or (N, C, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  labels: (N) where each value is 0 <= labels[i] <= C-1, or (N, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  
+  The loss for one sample, l_n, can caculated as follows
+      let p = Softmax(scores)
+      l_n = -sum(label_i * log(p_i)), where i is the index of classes.
+  or
+      l_n = -sum(weight_i * label_i * log(p_i)), if 'weights' is provided.
+  Finally, L is reduced:
+  L = ReduceSum(L), if reduction = 'sum';
+      ReduceMean(L), if reduction = 'mean';
+      L, if reduction = 'none'
+
```

Comment:
What about the K-dimensional loss? I don't quite understand how the above equations apply in that case.

Reply:
Is the same computation applied to each of of the k-dimensional points separately?

Reply:
It's the same as in NllLoss, SoftmaxCrossEntropy = Log(Softmax(scores)) + NllLoss(scores, labels).
L is the same shape as labels and if there is reduction, L is flattened and then sum or mean is performed and output is scalar.